docs/development_guide/osm_components_interactions.md

```diff
@@ -11,21 +11,22 @@ weight: 1
 ### Containers
```

Comment:
Since this is a duplicate lets delete this, please

Reply:
Initially I thought the this file is still used by official doc site. Seems that's not the case. I will delete it.